read-textile.ts

```diff
@@ -0,0 +1,206 @@
+import readline from 'readline';
+import util from 'util';
+import fs from 'fs';
+import Turndown from 'turndown';
+import * as Diff from 'diff';
+
+import {
+  textileToHtml,
+  textileToCheerio,
+  processTextile,
+  fullyParseTextile,
+} from './data/test-utilities/process-textile';
+import { preParserSteps } from './data/cli-functionality/parser-steps/pre-parser';
+import { textileWorkaroundSteps } from './data/cli-functionality/parser-steps/textile-workarounds';
+
+process.on('exit', () => fs.rmSync('./dist', { recursive: true, force: true }));
+
+const argsToRead = process.argv
+  .slice(2)
+  .filter((arg) => !arg.startsWith('--'))
+  .map((arg) => {
+    try {
+      const content = fs.readFileSync(arg);
+      return content.toString();
+    } catch (e) {
+      //Fine to ignore.
+    }
+    return arg;
+  })
+  .map((arg) => arg.replace(/\\n/g, '\n'));
+
+const flags: Record<string, boolean | string> = process.argv
+  .slice(2)
+  .filter((arg) => arg.startsWith('--'))
+  .reduce((acc, arg) => {
+    const [key, value] = arg.replace(/^--/, '').split('=');
+    return {
+      ...acc,
+      [key]: value ?? true,
+    };
+  }, {});
+
+const rl = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout,
+});
+
+const question = util.promisify(rl.question).bind(rl);
+
+const turndownService = new Turndown();
+turndownService.keep(['div', 'span']);
+
+const queryUser = async (query: string, outputHandler: (answer: string) => void) => {
+  rl.question(query, outputHandler);
+};
+
+const executeSequentially = (queries: string[], outputHandlers: ((answer: string) => void)[]) => {
+  queries.forEach(async (query, i) => await queryUser(query, outputHandlers[i]));
+};
+
+// TODO: concat postParser, htmlParser steps.
```

Comment:
is this TODO for this PR or for later work?

Reply:
Later work 😬 it's a bit boring. Post-v1, kind of a feature.